package_test.go

```diff
@@ -376,28 +376,24 @@ func (s *PackageSuite) TestIterGetErrors(c *C) {
 	}
 }
 
-type ScannerInt struct {
-	SI int
-}
+type ScannerInt int
```

Comment:
[Note] These changes to the tests for implementing scanner interfaces are not directly related to this PR but the test design makes a lot more sense with primitive types.